notebooks/tutorials/1_getting_started_lesson4/notebook.ipynb

```diff
@@ -0,0 +1,406 @@
+    "\n",
+    "    # Create filter for top 350 and bottom 350\n",
+    "    # assets based on their sentiment scores\n",
+    "    top_bottom_scores = (\n",
+    "        sentiment_score.top(350) | sentiment_score.bottom(350)\n",
+    "    )\n",
+    "\n",
+    "    return Pipeline(\n",
+    "        columns={\n",
+    "            'sentiment_score': sentiment_score,\n",
+    "        },\n",
+    "        # Set screen as the intersection between our filter\n",
+    "        # and trading universe\n",
+    "        screen=(\n",
+    "            base_universe\n",
+    "            & top_bottom_scores\n",
+    "        )\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, let's run our pipeline over a 3 year period to get an output we can use for our analysis. This will take ~1 minute."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Import run_pipeline method\n",
+    "from quantopian.research import run_pipeline\n",
+    "\n",
+    "# Specify a time range to evaluate\n",
+    "period_start = '2013-01-01'\n",
+    "period_end = '2016-01-01'\n",
+    "\n",
+    "# Execute pipeline over evaluation period\n",
+    "pipeline_output = run_pipeline(\n",
+    "    make_pipeline(),\n",
+    "    start_date=period_start,\n",
+    "    end_date=period_end\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In addition to sentiment data, we will need pricing data for all assets present in this period. We can easily get a list of these assets from our pipeline output's index, and pass that list to `prices` to get the pricing data we need:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Import prices function\n",
+    "from quantopian.research import prices\n",
+    "\n",
+    "# Get list of unique assets from the pipeline output\n",
+    "asset_list = pipeline_output.index.levels[1].unique()\n",
+    "\n",
+    "# Query pricing data for all assets present during\n",
+    "# evaluation period\n",
+    "asset_prices = prices(\n",
+    "    asset_list,\n",
+    "    start=period_start,\n",
+    "    end=period_end\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can use Quantopian's open source factor analysis tool, [Alphalens](https://www.quantopian.com/lectures/factor-analysis), to test the quality of our selection strategy. First, let's combine our factor and pricing data using get_clean_factor_and_forward_returns. This function classifies our factor data into quantiles and computes forward returns for each security for multiple holding periods. We will separate our factor data into 2 quantiles (the top and bottom half), and use 1, 5 and 10 day holding periods:"
```

Comment:
Give a quick explanation of what forward returns are, either here or after getting them.